Makefile

```diff
@@ -93,7 +93,8 @@ deb: build
 docs:
 	$(MAKE) -C docs docs
 
-gccgo: build
+gccgo:
+	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)-gccgo" -f "$(DOCKERFILE).gccgo" .
 	$(DOCKER_RUN_DOCKER) hack/make.sh gccgo
```

Comment:
We actually need this for power, so I guess there should be some logic "if target is gccgo and arch is amd64 use dockerfile.gccgo, if power use dockerfile.ppc64le" , or maybe rename the Dockerfile to .ppc64le.gccgo